packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -116,7 +116,7 @@ - (int64_t)configurePlayer:(FVPVideoPlayer *)player
   // Set up the event channel.
   FVPEventBridge *eventBridge = [[FVPEventBridge alloc]
       initWithMessenger:messenger
-            channelName:[NSString stringWithFormat:@"flutter.io/videoPlayer/videoEvents%@",
+            channelName:[NSString stringWithFormat:@"flutter.dev/videoPlayer/videoEvents%@",
```

Comment:
This was just an opportunistic change; I had mass-changed flutter.io to flutter.dev in most places a while ago, and apparently missed this channel name.